updatecli/updatecli.d/agent.yaml

```diff
@@ -0,0 +1,33 @@
+sources:
+  lastGithubRelease:
+    kind: githubrelease
+    spec:
+      owner: 'grafana'
+      repository: 'agent'
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      versionfilter:
+        kind: latest
+
+targets:
```

Comment:
I am not sure if it's on purpose but this manifest doesn't specify scm and action configuration so it will change files locally but not on the git repository

Reply:
For more context on why this is needed. When Updatecli interacts with a git repository, the repository is cloned in the directory `/tmp/updatecli/scm` the initial motivation was to avoid losing local changes as  Updatecli needs to create and reset branches. We wanted to play safe but that just created confusion with new Updatecli users so we have a very long running effort to support local git repositories.

Reply:
Thanks! This was some kind of WIP. 😊